### PR TITLE
fixes issue #72: re-worded paragraph for clairty per user feedback 

### PR DESCRIPTION
**Pull request by londoncalling**
Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by londoncalling** [comment]
@moxiegirl please review and merge, updates per feedback from ralabate@gmail.com

File: docs/mac/step_six.md [hunk 1]
```diff
@@ -36,12 +36,9 @@ If you don't already have a terminal open, open one now:
...
 5. Find the `IMAGE ID` for your `docker-whale` image.
 
-	In this example, the id is `7d9495d03763`.
+	 In this example, the id is `7d9495d03763`.
 
-	You'll notice that currently, the `REPOSITORY` shows the repository but not
-	the namespace for `docker-whale`. You need to include the `namespace` for
-	Docker Hub to associate it with your account.  The `namespace` is the same as
-	your account name.
+    Notice that the `REPOSITORY` shows the repo name (`docker-whale`), but not a namespace. For Docker Hub to associate this `docker-whale` image with your GitHub account, you need to rename it to `YOUR_GITHUB_USERNAME/docker-whale`. Your account name will indicate the `namespace` on GitHub for your image. We'll do this in the next step when we tag the image.
```
Comment (moxiegirl) [inline docs/mac/step_six.md]: Thanks @londoncalling. One wince: can "we" lose the "we" words --- cause like the queen isn't involved.   Meet my lovely pet, Peeve.

http://cdn.sheknows.com/articles/2011/06/pug.jpg

Actually, use of the third is generally considered impersonal and while the rest of this should be using we unless it crept in somewhere else.

File: docs/windows/step_six.md [hunk 2]
```diff
@@ -36,12 +36,9 @@ If you don't already have a terminal open, open one now:
...
 5. Find the `IMAGE ID` for your `docker-whale` image.
 
-	In this example, the id is `7d9495d03763`.
+	 In this example, the id is `7d9495d03763`.
 
-	You'll notice that currently, the `REPOSITORY` shows the repository but not
-	the namespace for `docker-whale`. You need to include the `namespace` for
-	Docker Hub to associate it with your account.  The `namespace` is the same as
-	your account name.
+	  Notice that the `REPOSITORY` shows the repo name (`docker-whale`), but not a namespace. For Docker Hub to associate this `docker-whale` image with your GitHub account, you need to rename it to `YOUR_GITHUB_USERNAME/docker-whale`. Your account name will indicate the `namespace` on GitHub for your image. We'll do this in the next step when we tag the image.
```
Comment (moxiegirl) [inline docs/windows/step_six.md]: We words! Ms. Peeve

**Comment by londoncalling** [comment]
@moxiegirl modified wording per your comments

**Comment by moxiegirl** [comment]
@londoncalling Thanks.  LGTM

**Pull request merged**